pkg/persistence/config/loader/config_loader.go

```diff
@@ -100,7 +104,7 @@ func LoadConfigFile(fs afero.Fs, context *LoaderContext, filePath string) ([]con
 
 	for _, cgf := range loadedConfigEntries {
 
-		result, definitionErrors := parseConfigEntry(fs, configLoaderContext, cgf.Id, cgf)
+		result, definitionErrors := parseConfigEntry(ctx, fs, configLoaderContext, cgf.Id, cgf)
```

Comment:
I am not happy that we need to pass the context in that deep, and would've considered looping over all results/errors in line 113 instead.

Not sure about it though.

Reply:
This would mean that we would only see success config events if all configs are valid, because `nil` is returned for the config slice if there was any config error.

Reply:
Slightly adjusted it a bit, so that the partial loaded data is returned and it's not needed anymore to pass the context that deep